corehq/apps/styleguide/templates/styleguide/bootstrap5/atoms/accessibility.html

```diff
@@ -108,50 +101,139 @@ <h5>A11y check</h5>
     </div>
   </div>
-  <p>
-    Throughout the documentation, when possible, we will call out A11y. For example, checking color contrast in
-    the buttons. We aim for AA compliant color contrast. You can check color contrast in tools such as
-    <a href="https://webaim.org/resources/contrastchecker/">https://webaim.org/resources/contrastchecker/</a>.
-  </p>
-  <p>
-    Try Chrome extension Funkify to experience some common barriers.
-  </p>
-  <p>
-    Read more about a11y at
-  </p>
+  <h2 id="testing">Quick ways to test web accessibility</h2>
   <ul>
-    <li><a href="https://a11yproject.com/">https://a11yproject.com/</a></li>
-    <li><a href="http://www.carbondesignsystem.com/guidelines/accessibility">http://www.carbondesignsystem.com/guidelines/accessibility</a></li>
-    <li><a href="https://www.w3.org/WAI/tutorials">https://www.w3.org/WAI/tutorials</a></li>
-    <li><a href="https://www.youtube.com/playlist?list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g">A11ycast videos</a></li>
+      <li><strong>Keyboard Navigation:</strong> Test if modified code is navigable by keyboard, an effective way
+        to ensure page navigability and screen reader visibility. More information and tips
+        <a href="https://webaim.org/techniques/keyboard/#testing">here</a>.</li>
+      <li><strong>Explore the Accessibility Tree:</strong> Use Chrome DevTools' accessibility feature to view the
+        accessibility tree of a page under development, understanding how page elements are included, visible, and
+        organized. This is especially helpful for understanding what a specific element is, how it’s labeled, and
+        how it’s defined in relation to other elements on the screen.</li>
+      <li><strong>Use a Screen Reader:</strong> Free options like VoiceOver for Mac, ORCA for Linux, and NVDA for
+        Windows can assist in testing accessibility. Warning: different screen readers can read the same webpage
+        differently so using one is a good launching point but it is also recommended to utilize different screen
+        readers in final testing.</li>
+      <li><strong>Check color contrast:</strong>We aim for AA compliant color contrast. You can check color
+        contrast in tools such as
+        <a href="https://webaim.org/resources/contrastchecker/">https://webaim.org/resources/contrastchecker/</a>.
+      <li><strong>WAVE Browser Extension:</strong> <a href="https://wave.webaim.org/">WAVE</a> which identifies Web
+      Content Accessibility Guideline (WCAG) errors</p></li>
+      <li>Keep in mind that this list is not exhaustive and covers only a portion of what makes a site accessible to
+        all users. Ensuring accessibility is an ongoing commitment to creating inclusive digital experiences for
+        everyone.</li>
   </ul>
-  <h2 id="screen-readers" class="pt-4">
-    Designing for Screen Readers
+
+  <h2 id="focusability" class="pt-4">
+    Focusability, Tabability, and Visibility
```

Comment:
I don't think it's technically a real word anyway, but accepted spelling seems to be with two b's
```suggestion
    Focusability, Tabbability, and Visibility
```

Reply:
huh I was debating the spelling because indeed this is not a word